plugins/vpp/ipsecplugin/vppcalls/dump_vppcalls.go

```diff
@@ -266,11 +268,12 @@ func (h *IPSecVppHandler) DumpIPSecSPD() (spdList []*IPSecSpdDetails, err error)
 
 			// Prepare meta and put to the metadata map
 			meta := &SpdMeta{
-				SpdID:   spdDetails.SpdID,
-				SaID:    spdDetails.SaID,
-				Policy:  spdDetails.Policy,
-				Bytes:   spdDetails.Bytes,
-				Packets: spdDetails.Packets,
+				SpdID:     spdDetails.SpdID,
+				SaID:      spdDetails.SaID,
+				Policy:    spdDetails.Policy,
+				Bytes:     spdDetails.Bytes,
+				Packets:   spdDetails.Packets,
+				NPolitics: numPolitics,
```

Comment:
It does not make much sense to put number of policies inside each policy for SPD.